servlet-dist-legacy/src/distribution/resources/docs/licenses/licenses-merged.xsl

```diff
@@ -0,0 +1,43 @@
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
```

Comment:
This must be skipped by the checkstyle plugin, but we shouldn't use tab delimiters :)

Reply:
@jamezp You're right, I'm going to remove the tabs here and from dist-legacy/src/distribution/resources/docs/licenses/licenses-merged.xsl were are also incorrect.

Reply:
Fixed.